content/zh/docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -458,36 +458,36 @@ kubectl apply -f https://docs.projectcalico.org/v3.1/getting-started/kubernetes/
 ```
 
 {{% /tab %}}
-
-
-
+<!--
```

Comment:
@aanm Thanks for your submission.
But I think `<!--` should go under `{{% TAB name="Cilium" %}}` to preserve the structure of the document.

Reply:
@SataQiu thanks for the review. Done!